charts/kong/templates/_helpers.tpl

```diff
@@ -355,6 +355,12 @@ The name of the service used for the ingress controller's validation webhook
 
 {{- end -}}
 
+{{- define "kong.userDefinedVolumes" -}}
+  {{- if .Values.deployment.userDefinedVolumes }}
+  {{- toYaml .Values.deployment.userDefinedVolumes }}
+  {{- end }}
```

Comment:
Indentation here also looks wrong--IIRC `toYaml` doesn't play nicely with in-line indents and will indent the first line only.

These should probably all have no indent, similar to what we have for the other volume helpers, e.g. https://github.com/Kong/charts/blob/kong-1.15.0/charts/kong/templates/_helpers.tpl#L366-L370. Not sure, but CI should confirm--if it's wrong, the result will be a rendering error.

Reply:
fixed